test/testswmr.pwsh.in

```diff
@@ -11,8 +11,8 @@
 # Tests for the swmr feature.
 
 $srcdir  = '@srcdir@'
-$bindir  = '@bindir@'
-$testdir = '@testdir@'
+$utilstestdir=@H5_UTILS_TEST_BUILDDIR@
+$testdir=@H5_TEST_BUILDDIR@
```

Comment:
It might be worth noting that we have to set testdir because CMake puts the binaries in different places than the Autotools. A comment here (and copied to the other places where the paths are set up) would help people who don't build/test with CMake from undoing this.